src/Microsoft.Data.SqlClient/netcore/src/.editorconfig

```
@@ -10,3 +10,6 @@ csharp_style_implicit_object_creation_when_type_is_apparent = false
 
 # IDE0063: Use simple 'using' statement
 csharp_prefer_simple_using_statement = false
+
+# SYSLIB0039: Type or member is obsolete
+dotnet_diagnostic.SYSLIB0039.severity = none
```

I'd rather keep the report in the error list and unblock the build.
```suggestion
dotnet_diagnostic.SYSLIB0039.severity = suggestion
```